src/Compilers/CSharp/Test/Emit/Emit/DesktopStrongNameProviderTests.cs

```diff
@@ -1,10 +1,9 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
+using System.Collections.Immutable;
+using System.IO;
 using Microsoft.CodeAnalysis.CSharp.Test.Utilities;
-using Microsoft.CodeAnalysis.Test.Utilities;
 using Roslyn.Test.Utilities;
-using System;
-using System.IO;
 using Xunit;
 using static Roslyn.Test.Utilities.SigningTestHelpers;
```

Comment:
sort usings

Reply:
This is a comment on the "old" file of the diff.